src/ml_gym/error_handling/exception.py

```diff
@@ -91,6 +91,11 @@ class OptimizerNotInitializedError(Exception):
     pass
 
 
+class LRSchedulerNotInitializedError(Exception):
+    """Raised when we want to run an operation on a lr scheduler, which was not instantiated, yet."""
+    pass
```

Comment:
rather than ending with a pass can we try to log the exception or is it handled somewhere else.

Reply:
The exception is being raised and then handled in the code, where the exception ocurred.